lib/ruby-push-notifications/mpns/mpns_result.rb

```diff
@@ -0,0 +1,182 @@
+
+module RubyPushNotifications
+  module MPNS
+    # Class that encapsulates the result of a single sent notification to a single
+    # Device URL
+    # (https://msdn.microsoft.com/en-us/library/windows/apps/ff941100%28v=vs.105%29.aspx)
+    class MPNSResult
+      # @return [String]. Receiver MPNS device URL.
+      attr_accessor :device_url
+
+      # @private X-NotificationStatus HTTP Header string
+      X_NOTIFICATION_STATUS  = 'x-notificationstatus'
```

Comment:
Why lower case? According to docs they are camel case, aren't they?
https://msdn.microsoft.com/en-us/library/windows/apps/ff941100(v=vs.105).aspx#BKMK_CustomHeaders
Can you test it on real device to see how are they exactly?